test/androidTest/java/org/thoughtcrime/securesms/espresso/ConversationHelper.java

```diff
@@ -89,6 +93,36 @@ public ConversationHelper sendReaction(int position, int emoji) {
         return this;
     }
 
+    public ConversationHelper sendReply(int position, String message) {
+
+        this.unselectMessage();
+        this.selectMessage(position);
```

Comment:
Following the pattern in the rest of the file, could you add a line between these two method calls